article/app/views/fragments/recipeArticleBody.scala.html

```diff
@@ -167,6 +167,9 @@ <h1 itemprop="name">@recipe.data.title</h1>
                 <li class="recipe__article--list-title">Method</li>
                 @recipe.data.steps.zipWithIndex.map{case(step, i) => <li><span class="number">@{i + 1}</span>@step</li> }
             </ul>
+            <div class="recipe__content--feedback">
+              <p class="recipe__content--feedback-text">You’re seeing a new design for our recipes and we would really appreciate your feedback. <a target="_blank" class="recipe__content--feedback-link" data-link-name="new recipe: form" href="https://docs.google.com/a/guardian.co.uk/forms/d/e/1FAIpQLSdMmHTu5RlS9oI-6CZctYxXjMwVGnzfV1SfYd-OYDsJvc2bJw/viewform?usp=pp_url&entry.1145678620=@articleModel.item.trail.headline">Leave feedback</a></p>
+            </div>
```

Comment:
If you are using `target="_blank"` please always add `rel="noopener noreferrer"` because of https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/.